Sources/SpotifyWebAPI/Authorization/SpotifyAuthorizationManager.swift

```diff
@@ -103,4 +103,14 @@ public protocol SpotifyAuthorizationManager: Codable {
     /// Calling this method should cause `didDeauthorize` to emit a signal.
     func deauthorize() -> Void
     
+	/**
+	  Used internally by the authorization managers in this library.
+	  Do not implement it yourself. A default implementation is provided
+	  which does nothing.
+	  */
```

Comment:
This comment is slightly different from the version on `proxy-tokens-server`:

https://github.com/Peter-Schorn/SpotifyAPI/blob/aad995f50bfd194e7895aaaab6636dec5be2cc4c/Sources/SpotifyWebAPI/Authorization/SpotifyAuthorizationManager.swift#L106-L111

This is why I'm concerned about merging directly into master. What if we need to make more changes to `proxy-tokens-server`? And if they're breaking changes, then that makes things more complicated..

Reply:
More changes are probable, but every bit of extracted changes shrink the PR and therefore its complexity. I think there's no arguing the fact that this function needs to be part of the protocol to facilitate subclasses and generics in general, correct?

Reply:
Which is semantically unrelated to the PR, which just happens to require it as a pre-requisite.

Reply:
Ok, I will merge this change if you update the comment to match the one on `proxy-tokens-server`.

Reply:
Done